internal/webhook/v1alpha1/service_account_utils.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
do we really need service_account_utils.go? why not keep it together with defaulter? (cohesion should allow that)